forgerock-authenticator/src/main/java/org/forgerock/android/auth/PushNotification.java

```diff
@@ -357,17 +357,24 @@ public final void accept(String title,
         if (this.pushType == PushType.BIOMETRIC) {
             final PushNotification pushNotification = this;
             BiometricAuth biometricAuth = new BiometricAuth(title,
-                    subtitle, allowDeviceCredentials, activity, new BiometricAuthCompletionHandler() {
+                    subtitle, allowDeviceCredentials, activity, new AuthenticationCallback() {
```

Comment:
for this change , after we merge,  @spetrov we need to test the AuthenticatorApp to verify we break anything

Reply:
👍🏻

Reply:
Will add another ticket for the test.